### DISCUSSION TITLE
Friesach: How turn off noise?

### DISCUSSION THREAD

**Discussion by jason-barba** [GrandOrgue/grandorgue#1622] [Help] [answered]
Asking for a friend: How do I turn of the noise sound in Friesach? I cannot find any way to do this.

//Jens

**Comment by kerkovits** [comment, marked as answer]
1. Load the organ
2. Sound / Midi → Organ settings → Select "Noises" in the left panel.
3. Set Sample channels in the right panel to "Do not load"
4. Apply
5. OK
6. Save
7. Reload

**Reply by jason-barba** [reply]
Thank you. My friend is most grateful ;-)

**Reply by powerinpete** [reply]
Also, if you want to turn off the noises from the keyboard actions (which aren't all that loud in Freisach) these are in a different place. Keyboard noises for all manuals/pedal are found inside the "Solowerk" section just above the Noises. 

You will have to expand that section and repeat the procedure above for the 8 different Key action attack and release noises listed.

**Comment by ClarkeBrunt** [comment]
Just in case of interest, when I was new to GO and the Friesach organ,  I posted some observations about the Friesach set here #1407 . I go on about other stuff there, but also note what are maybe mistakes in the sample set concerning the noises. e.g. some of the noises are turned off by 'General Cancel' (there isn't a visible GC, but of course you can find one in the panels which GO adds to all sets), and there is then no obvious way to turn them back on. And some of the noises maybe allocated to the wrong windchest (as mentioned above, you wouldn't expect noises to be part of the 'Solowerk', rather than 'Noises'.

This isn't really appropriate stuff to go in the help file, as it's specific to the Friesach organ.
